mobile/cmp/button/MultiZoneMapperButton.ts

```diff
@@ -0,0 +1,44 @@
+/*
+ * This file belongs to Hoist, an application development toolkit
+ * developed by Extremely Heavy Industries (www.xh.io | info@xh.io)
+ *
+ * Copyright © 2023 Extremely Heavy Industries Inc.
+ */
+import {hoistCmp, useContextModel} from '@xh/hoist/core';
+import {MultiZoneGridModel} from '@xh/hoist/mobile/cmp/multiZoneGrid';
+import {button, ButtonProps} from '@xh/hoist/mobile/cmp/button';
+import {Icon} from '@xh/hoist/icon';
+import {withDefault} from '@xh/hoist/utils/js';
+import '@xh/hoist/mobile/register';
+
+export interface MultiZoneMapperButtonProps extends ButtonProps {
+    multiZoneGridModel?: MultiZoneGridModel;
```

Comment:
nit: wonder if this could just be `gridModel`.  Let the type do the work

Reply:
ah -- see it is actually not a `GridModel` superclass. Not sure about this.

Reply:
I renamed to `zonedGridModel?: ZonedGridModel`